includes/wikia/DefaultSettings.php

```diff
@@ -1833,6 +1833,12 @@
  */
 $wgUseGoogleCloudStorage = false;
 
+/**
+ * https://wikia-inc.atlassian.net/browse/SER-3033
+ * If enabled, a multi write will be used for storing files.
+ */
+$wgUseGcsMigrationStorage = false;
```

Comment:
I would use a bucket prefix here, since it will be easier to control than setting flags on multiple wikis. We can set a prefix of out "global wiki" and retrieve it like this:

    \WikiFactory::getVarValueByName("wgUseGcsMigrationBucketPrefix", static::DEFAULT_WIKI_ID );